Makefile.am

```diff
@@ -51,7 +51,9 @@ nobase_dist_config_DATA = \
         test_configs/udp/quick.test \
         test_configs/udp/simple.test \
         test_configs/verbs/all.test \
-        test_configs/verbs/quick.test
+        test_configs/verbs/quick.test \
+	test_configs/usnic/all.test \
```

Comment:
I don't know why this looks wrong in the diff. The line above it is 8-spaces, I think we should just stick to 8-space hard tabs.